src/scripts/main.js

```diff
@@ -1,3 +1,29 @@
 'use strict';
 
-// write your code here
+const body = document.querySelector('body');
+const logo = document.querySelector('.logo');
+
+const promise1 = new Promise(resolve => {
+  logo.addEventListener('click', () => {
+    resolve();
+  });
+});
+
+const promise2 = new Promise(( resolve, reject) => {
+  setTimeout(() =>
+    reject(new Error()), 3000);
+});
+
+promise1.then(() => {
+  body.insertAdjacentHTML('beforeend',
+    `<div class="message">
+      Promise was resolved!
+    </div>`);
+});
+
+promise2.catch(() => {
```

Comment:
<img width="354" alt="image" src="https://user-images.githubusercontent.com/102411130/223950749-0752f8a3-b6f3-4b39-b1a2-1e838b10c79e.png">
check this requirement point